lib/date.nit

```diff
@@ -0,0 +1,254 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date and Time manipulation
+# The class Date has the basic operations to use only the date data
+# The class Time has the basic operations to use only the time data
+# The class DateTime extend the class Date and Time to use the two of date and time data
+#
+# 	var p_date = new Date.set(2014, 01, 09)
+#	var auto_date = new Date.today ==> set the current date 
+#	var auto_time = new Time.now ==> set the current time
+#	var date_time = new DateTime.today ==> set the current date and time
```

Comment:
Why don't you use the default constructor for set `now` / `today`?

~~~nit
new Date # returns today date
new Time # returns now time
~~~